common/mod/ngv/vascouplingB.mod

```diff
@@ -180,14 +181,16 @@ BREAKPOINT{
     : PERIVASCULAR SPACE
     :kp = kp + dt*((1/VRpa)*j_BK+(1/VRps)*(j_KIR)-Rdecay*(kp-kpmin)) :Perivascular K+
     :kp = kp + dt*((1/VRpa)*j_BK+(1/VRps)*(j_KIR)-Rdecay*1000*(kp-kpmin))
+
+:RESTORE NEXT
     kp = kp + dt*((1/VRpa)*j_BK*0.0001+(1/VRps)*(j_KIR)-Rdecay*1000*(kp-kpmin))
     :kp = kp + dt*((1/VRpa)*j_BK*0.0001-Rdecay*1000*(kp-kpmin))
 
     : Membrane potential of SMC cell. Added mechanism to come back to resting potential, otherwise stays hyperpolarized
     : Via activation of KIR channel
     :Vi = Vi + dt*(1/Cmu)*(I_KIR)*pow(10,7) - dt*keet*5*pow(10,-2)*(Vi+30): with inactivation
     :Vi = Vi + dt*(1/Cmu)*(I_KIR)*pow(10,7)  - dt*keet*5*pow(10,-2)*(Vi+30): with inactivation
-    Vi = Vi + dt*(1/Cmu)*(I_KIR)*pow(10,7)  - dt*eete*1.2-0.001*(Vi+30)
+    Vi = Vi + dt*(1/Cmu)*(I_KIR)*pow(10,7)  - 0*dt*eete*1.2-0.001*(Vi+30)
```

Comment:
Hello @dkeller9 , is it times 0 `0*dt*eete...` on purpose ?

Reply:
Yes, this pathway was causing issues so I disabled it.